packages/rest/src/rest.server.ts

```diff
@@ -600,16 +591,7 @@ export class RestServer extends Context implements Server, HttpServerLike {
    */
   async stop() {
     // Kill the server instance.
-    const server = this._httpServer;
-    return new Promise<void>((resolve, reject) => {
-      server.close((err: Error) => {
-        if (err) {
-          reject(err);
-        } else {
-          resolve();
-        }
-      });
-    });
+    return this._httpServer.stop();
```

Comment:
Now that you are directly returning the promise provided by `stop`, this function no longer needs to be async. 

```diff
-   async stop() {
+  stop() {
        // Kill the server instance.
```

Reply:
Will await `this._httpServer.stop()`, instead.